### PR TITLE
Remove reset/delete account progress animations and change color/increase size of buttons 

### LINKED ISSUE #11382: Update UX on reset progress and delete account buttons
Refer https://github.com/FreeCodeCamp/FreeCodeCamp/pull/10824

### PR DESCRIPTION
**Pull request by johnnyqbui**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #11382

#### Description

Remove reset progress and delete account animations, change color of success button to green, and increase button size tap targets

### CODE REVIEW

File: client/less/main.less [hunk 1]
```diff
@@ -1145,6 +1145,53 @@ and (max-width : 400px)  {
...
+    padding-left: 10px;
+    margin-top: 22px;
+  }
+
+  .modal-header {
+    text-align: center;
+    padding: 5px 20px;
+  }
+
+  .modal-footer {
```
Comment (atjonathan) [inline client/less/main.less]: Add a space between each class's styles declared inside of `.modal-footer`.

**Review by ghost — CHANGES_REQUESTED** [review]

File: client/less/main.less (hunk 1 above)
Comment (johnnyqbui) [inline client/less/main.less]: @atjonathan I've made the changes.

Comment (johnnyqbui) [inline client/less/main.less]: @atjonathan I was trying to squash the commits, but I think I messed up.

**Comment by dhcodes** [comment]
LGTM. 

@atjonathan Is the following behavior intended?
When canceling the modal for deleting the account, it goes to the about page.
When canceling the modal for resetting progress, it goes back to the settings page.

That may be outside the scope of this PR.

**Review by ghost — APPROVED** [review]

**Comment by atjonathan** [comment]
@dhcodes 
> When canceling the modal for deleting the account, it goes to the about page.

That behavior should be changed. The other seems fine. I opened a new issue (#11922), as this PR is irrelevant. 

@johnnyqbui, **LGTM 👍**
Thanks for contributing, loving the UI improvements 💪

**Pull request merged**